system/emergency_handler/src/emergency_handler/emergency_handler_core.cpp

```diff
@@ -396,7 +394,7 @@ autoware_system_msgs::msg::HazardStatus EmergencyHandler::judgeHazardStatus()
           "heartbeat_driving_capability is timeout"));
     }
 
-    if (heartbeat_is_state_timeout_->isTimeout()) {
+    if (heartbeat_is_state_timeout_->isTimeout() && !is_in_heartbeat_timeout_ignore_state) {
```

Comment:
and one more small thing, writing `is_in_heartbeat_timeout_ignore_state` before `isTimeout()` seems better.

```cpp
    if (!is_in_heartbeat_timeout_ignore_state && heartbeat_is_state_timeout_->isTimeout()) {
```